src/Core/Geometry/StandardAttribNames.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <Core/RaCore.hpp>
+
+namespace Ra {
+namespace Core {
+namespace Geometry {
+
+/// List of standard vertex attributes.
+/// Corresponding standard vertex attribute names are obtained with g_attribName
+/// Information which is in the mesh geometry
+enum MeshAttrib : uint {
```

Comment:
```suggestion
/// \brief List of standard vertex attributes.
/// Corresponding standard vertex attribute names are obtained with #getAttribName
enum MeshAttrib : uint {
```

Reply:
Can I accept the suggestion or should it be a Radium contributor to accept ?

Reply:
I have a commit suggestion button in my side